pytorch_lightning/plugins/ddp_plugin.py

```diff
@@ -63,8 +64,8 @@ def configure_ddp(self, model, device_ids):
         self._ddp_kwargs["find_unused_parameters"] = self._ddp_kwargs.get(
             "find_unused_parameters", True
         )
-        model = LightningDistributedDataParallel(
-            model,
+        model = DistributedDataParallel(
+            module=LightningDistributedWrapper(model),
```

Comment:
the docstring above needs to be updated for the return type

Reply:
done